setup.py

```diff
@@ -11,8 +11,10 @@
       url='http://github.com/fitbenchmarking/fitbenchmarking',
       license='GPL-3.0',
       packages=find_packages(),
-      install_requires=['docutils', 'scipy>=0.17,<1.3', 'bumps', 'sasmodels', 'lxml'],
+      install_requires=['docutils', 'numpy<1.17', 'matplotlib<3.0',
+                        'scipy>=0.17,<1.3', 'bumps', 'sasmodels', 'lxml', 'h5py'],
```

Comment:
`scipy=0.17` fails for me on Ubuntu (issue #128) -- I'd like to see this be `scipy>=0.18`

Reply:
Done!

Reply:
Cool -- this now fixes #128 (adding this in the hope GitHub will close that when this is merged)

@Anders-Markvardsen @AnthonyLim23 : please shout if there is a hidden reason why we should be supporting scipy 0.17